PHPblog.php

```diff
@@ -8,27 +14,23 @@
 	<body>
 		<h1 class='header'> Lindsay's Blog </h1>
 		<h2 class='subtitle'>WORDS FROM A GIRL LEARNING HOW TO CODE </h1>
-		<div class='blogpost'; style=background-color:#bc1a1a;>
-			<h3>Halfway Done with College </h3>
-			<p class='white'>A look back on sophomore year</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=0'>Read on</a></p>
-		</div>
-
-		<div class='blogpost'; style=background-color:#153466;>
-			<h3>My Summer in St. Louis</h3>
-			<p class='white'> Code, run, sleep.</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=1'>Read on</a></p>
-		</div>
-
-		<div class='blogpost'; style=background-color:#8fd7e8;>
-			<h3>My opinion on carrots</h3>
-			<p class='white'> The inside scoop on this special veggie</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=2'>Read on</a></p>
-		</div>
 
+<?php
 
+function PostABlock($blocknumber){
+	$sorter=getSorter();
+	$item = $sorter[$blocknumber];
+		echo "<div class='blogpost'; style='background-color:$item[BlockColor]'>";
+		echo "<h3>$item[ArticleTitle]</h3>";
+		echo "<p class='white'>$item[Subtitle]</p>";
+		echo "<p class='white'><a href=$item[ReadOn]>Read on</a></p>";
+}
 
+PostABlock('3');
```

Comment:
Ideally you'd have code that determines what posts to print so you don't have to hard-code these three specific calls.

A simple example of this would be to just loop through the `$sorter` array and print a block for each post. If you don't want to print everything, you could make a field in that array that says whether or not you want an item to be printed, and look at that before printing a field.

This might seem pointless right now (what's the difference between hard-coding something in $sorter vs. hard-coding it here. the reason it matters is that the next step is to move everything from $sorter into the database. The same way you want your html and css separated, you want your code and content separated. You want to be able to write a new blog post without writing any new code, and right now that wouldn't be possible because you'd need to updating this section here.